src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -174,6 +176,10 @@ export /* istanbul ignore next */ function PipelineRuns(props) {
     rerunPipelineRun(pipelineRun);
   }
 
+  function start(pipelineRun) {
```

Comment:
Looks like this isn't actually required since it's just passing the params directly to `startPipelineRun` so we could just call that directly. My guess is that we used to have more logic in the `rerunPipelineRun` and never removed the function after changing it.

Reply:
removed both methods